src/io/PWIZSpectrumCollection.cpp

```diff
@@ -184,11 +188,14 @@ bool PWIZSpectrumCollection::parse() {
       carp(CARP_FATAL, "boost::bad_lexical_cast occured while parsing spectrum.\n"
                        "Do your spectra contain z-lines?");
     }
+
+    int curr_ms_level = spectrum->cvParam(pwiz::msdata::MS_ms_level).valueAs<int>();
+    carp(CARP_DEBUG, "ms_level=%d\t", curr_ms_level );
+
     // skip if no peaks or not ms2
-    if (spectrum->defaultArrayLength < 1 ||
-        spectrum->cvParam(pwiz::msdata::MS_ms_level).valueAs<int>() != 2) {
-      continue;
-    }
+    // if (spectrum->defaultArrayLength < 1 || spectrum->cvParam(pwiz::msdata::MS_ms_level).valueAs<int>() != 2) { continue; }
```

Comment:
Why did you comment this out?

Reply:
Otherwise, PWIZParser only read ms2 spectra but not ms1 spectra